postgres.go

```diff
@@ -25,6 +28,93 @@ type DBConf struct {
 	clientKey  string
 }
 
+// Basebackup function:
+// - gets an identical copy of the pg database (pg_data)
+// - verifies the backup
+// - tars the copy
+// - gets the key and encrypts the tar file
+// - compresses the encrypted file
```

Comment:
compression should be done before encryption, as encrypted files compress way worse!

Reply:
Actually this is the order that is doing it, first compress and then encrypt. So when you do the write it does it in the opposite way than what you read in the function. You can check also the `dump` function which is done in the same way and because it was also strange to me I double checked by changing the order and the final file watch much bigger. I updated the comments to be in the correct order.